longshot/main.tf

```diff
@@ -37,49 +37,23 @@ provider "null" {
   version = "~> 2.1"
 }
 
-resource "heroku_pipeline" "longshot" {
-  name = "longshot"
-}
-
 module "longshot-footlocker" {
   source = "../applications/longshot"
 
   name        = "longshot-footlocker"
-  domain      = "footlockerscholarathletes.com"
-  pipeline    = "${heroku_pipeline.longshot.id}"
   environment = "production"
-  deprecated  = true
```

Comment:
Do we still want this deprecated flag?

Reply:
It had originally been a way to tear down the database (since you first need to delete users & disable "deletion protection" by "deprecating" that resource before actually deleting it). At this point, it's no longer being used.